aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/dto/ApplicationCommonDetails.java

```diff
@@ -0,0 +1,14 @@
+package com.castsoftware.aip.console.tools.core.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+public class ApplicationCommonDetails {
+
+    public List<ApplicationCommonDetailsDto> applicationCommonDetailsDtoSet = new ArrayList<>();
```

Comment:
Use Builder.Default

Reply:
Are we doing this for immutability and to be thread safe?